compiler/rustc_session/src/session.rs

```diff
@@ -1725,13 +1723,13 @@ impl EarlyErrorHandler {
 
     #[allow(rustc::untranslatable_diagnostic)]
     #[allow(rustc::diagnostic_outside_of_impl)]
-    pub fn early_note(&self, msg: impl Into<DiagnosticMessage>) {
+    pub(crate) fn early_note(&self, msg: impl Into<DiagnosticMessage>) {
         self.handler.struct_note_without_error(msg).emit()
     }
 
     #[allow(rustc::untranslatable_diagnostic)]
     #[allow(rustc::diagnostic_outside_of_impl)]
-    pub fn early_help(&self, msg: impl Into<DiagnosticMessage>) {
+    pub(crate) fn early_help(&self, msg: impl Into<DiagnosticMessage>) {
```

Comment:
Someone out-of-tree might need these.

Reply:
I will make these `pub` again: `early_help`, `early_note`, `abort_if_error_and_set_error_format`.

I will also change this one from `pub(crate)` to `pub`, for consistency: `early_struct_error`.